pegasus/sites.v3/code.org/views/help/help_page_bring_cs_to_school.haml

```diff
@@ -12,13 +12,13 @@
 
 %div.editorial-cards-wrapper
   - bring_cs_to_school_index.times do |index|
-    %div.flex-container.align-content-baseline.wrap
-      %figure
-        %img{src: bring_cs_to_school_img_src[index], alt: ""}
+    .flex-container.justify-space-between.align-content-baseline.wrap
```

Comment:
I assume these changes are relevant too, yes?

Reply:
Correct! It was just the CSS linter changes that aren't.